citation-demo/work/master/model.json

```diff
@@ -0,0 +1,53 @@
+[
```

Comment:
model.json should be commited in `citation-demo-content`, `citation-demo/work` should be ignored